src/server/rpc/libs/RPC

```diff
@@ -8,3 +8,4 @@ weather.xml	Weather API
 pubsub.xml	PubSub
 kvstore.xml	Key-Value Store
 battleship.xml	Battleship
+Traffic.xml	Traffic
```

Comment:
change `Traffic.xml` to `traffic.xml` - this will break on linux which is case sensitive